app/aws/solutions-architect-associate/domain4/page.css

```
@@ -0,0 +1,386 @@
+.domain4-page .mermaid-wrap {
+    margin: 8px 0 32px;
+    background: var(--bg-elevated);
+    border: 1px solid var(--border-soft);
+    border-radius: 12px;
+    padding: 24px;
+    display: flex;
+    justify-content: center;
+}
+
+/* Pie Chart 配色オーバーライド（鮮やかなカラーの維持） */
+.domain4-page .mermaid-wrap :global(.pieCircle) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)